packages/docs/site/docs/blueprints/02-using-blueprints.md

```diff
@@ -26,7 +26,24 @@ For example, to create a Playground with specific versions of WordPress and PHP
 ```
 
 And then you would go to
-`https://playground.wordpress.net/#{"preferredVersions": {"php":"7.4", "wp":"6.5"}}`.
+`https://playground.wordpress.net/#{"preferredVersions":{"php":"7.4","wp":"6.5"}}`.
+
+:::tip
+In Javascript, you can get a compact version of any blueprint json with [`JSON.stringify`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify) and [`JSON.parse`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse)
```

Comment:
I believe we should spell "json" as "JSON" according to the style guide: https://make.wordpress.org/docs/style-guide/formatting/filenames/